geometry2/561d66b/561d66b.bug

```diff
@@ -94,4 +94,4 @@ fault-codes:
   - BDO:DEPENDENCY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:LOSS-OF-FUNCTIONALITY
```

Comment:
We agreed at the meeting that SYSTEM:BEHAVING-INCORRECTLY seems more appropriate here